lib/event_sourcery/rspec/event_store_shared_examples.rb

```diff
@@ -1,66 +1,59 @@
 RSpec.shared_examples 'an event store' do
-  let(:aggregate_id) { SecureRandom.uuid }
+  TestEvent2 = Class.new(EventSourcery::Event)
```

Comment:
Is it okay that our internal concepts leak into an open-sourced gem as event names? What if for consistency purpose we pick a domain for our events? I.e. to-do app, blog, user management, etc. Something that should be  easy to understand for external contributors. What do you think @twe4ked?